Modules/_ssl.c

```diff
@@ -1515,7 +1515,13 @@ _get_crl_dp(X509 *certificate) {
         STACK_OF(GENERAL_NAME) *gns;
 
         dp = sk_DIST_POINT_value(dps, i);
+        if ((dp == NULL) || (dp->distpoint == NULL)) {
```

Comment:
Is `dp == NULL` actually possible here?

Reply:
OpenSSL doesn't check for ``dp == NULL``, so it's not necessary.